assets/js/base/components/product-list/product-sort-select/index.tsx

```diff
@@ -2,19 +2,20 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import SortSelect from '@woocommerce/base-components/sort-select';
-
 /**
  * Internal dependencies
  */
 import './style.scss';
+import { ProductSortSelectProps } from '../types';
 
-const ProductSortSelect = ( { onChange, readOnly, value } ) => {
+const ProductSortSelect = ( {
+	onChange,
+	value,
+}: ProductSortSelectProps ): JSX.Element => {
 	return (
 		<SortSelect
 			className="wc-block-product-sort-select wc-block-components-product-sort-select"
-			name="orderby"
```

Comment:
removed invalid `name` property being passed to SortSelect component. This is not defined or used in `SortSelect`